.github/workflows/ci.yml

```diff
@@ -2,12 +2,10 @@ name: CI
 
 on:
   pull_request:
-    branches: [development]
     paths-ignore:
       - '*.md'
 
   push:
-    branches: [development]
```

Comment:
I think we want to keep this. I think we only want the CI to run for push events to a pull request, but not on a push to any branch without PR.

Reply:
That makes sense, reverted in: e1bcaf6